src/components/Dialog/Dialog.st.css

```diff
@@ -39,7 +44,23 @@
     left: 50%;
     transform: translate(-50%, -30%);
 
-    background: #fff;
+    background: value(DefaultBackgroundColor);
+}
+
+.closeButtonWrapper {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+}
+
+.closeIconButton {
+    -st-mixin: skin-line(
+            IconColor value(DefaultCloseButtonColor)
+    );
+}
```

Comment:
this can be transformed to a `skin-fixed-colors`
`-st-mixin: skin-fixed-colors(IconColor value(CloseButtonColor))`

and the `skin-fixed-colors` variation will be an overridable parameter with a fallback to `#000000`

In my head, it's like propagating a prop const `Dialog = ({skin}) => <div> ... <IconButton skin={skin}/> ... </div>`